### PR TITLE
add deploy config for new Prow POC cluster

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
The PR contains the source of truth deployment configuration files for the new Prow POC cluster deployed as part of the infra migration, from Individual account to group ES account.

### CODE REVIEW

**Comment by Priyankasaggu11929** [comment]
@AndyTauber @rajaskakodkar please review!

File: config/jobs/build-images/build.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 postsubmits:
   rajaskakodkar/tanzu-test-infra:
   - name: build-test-image
+    cluster: prow-service-trusted
```
Comment (rajaskakodkar) [inline config/jobs/build-images/build.yaml]: I think we should run this on build cluster

File: config/jobs/tanzu-test-infra/postsubmit-trusted.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      volumeMounts:
+      - name: github-token
+        mountPath: /etc/github-token
+        readOnly: true
+#      - name: ssh
+#        mountPath: /root/.ssh
+    volumes:
+    - name: github-token
+      secret:
+        secretName: github-token
```
Comment (rajaskakodkar) [inline config/jobs/tanzu-test-infra/postsubmit-trusted.yaml:53]: this should be github personal access token right?

File: config/prow/cluster/kubernetes-external-secrets_sa.yaml [hunk 3]
```diff
@@ -3,8 +3,7 @@ kind: ServiceAccount
 metadata:
   annotations:
     eks.amazonaws.com/audience: sts.amazonaws.com
-    eks.amazonaws.com/role-arn: arn:aws:iam::609817409085:role/prow-ecr
+    eks.amazonaws.com/role-arn: xyz
```
Comment (rajaskakodkar) [inline config/prow/cluster/kubernetes-external-secrets_sa.yaml:6]: shouldn't we give this the correct role name?

**Review by rajaskakodkar — CHANGES_REQUESTED** [review]
quickly skimmed through this and added few comments

File: config/jobs/build-images/build.yaml (hunk 1 above)
Comment (Priyankasaggu11929) [reply config/jobs/build-images/build.yaml]: ack. Updated!

File: config/prow/cluster/kubernetes-external-secrets_sa.yaml (hunk 3 above)
Comment (Priyankasaggu11929) [reply config/prow/cluster/kubernetes-external-secrets_sa.yaml:6]: I was thinking if we could update this one with the external secrets PR  #62 getting merged?

Comment (rajaskakodkar) [reply config/prow/cluster/kubernetes-external-secrets_sa.yaml:6]: that works as well. Is #62 updated? Is it in a state to be reviewed?

**Review by rajaskakodkar — APPROVED** [review]

**Comment by rajaskakodkar** [comment]
@Priyankasaggu11929 crier service is not present in this PR. Can you please check?

**Comment by Priyankasaggu11929** [comment]
>  crier service is not present in this PR. Can you please check?

@rajaskakodkar, addressed ^ in #64

**Pull request merged**